### PR TITLE
Added messages constants and some defaults

### LINKED ISSUE #21: Add message constants
Add message constants related to each module or component in their respective directories along with shared messages. Messages meaning text to be displayed for a certain event or for a certain action.

### PR DESCRIPTION
**Pull request by pranjulps3**
Closes #21

### CODE REVIEW

File: src/app/shared/services/github.provider.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { ɵa as BaseLoginProvider } from 'angularx-social-login';
```
Comment (Samy-33) [inline src/app/shared/services/github.provider.ts]: Garbage character?

File: src/app/shared/messages/auth.messages.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// TODO: Add message codes [Error] [Success] [Info] etc
+
+export const LoginSuccessMsg = "Successfully Logged In!";
```
Comment (Samy-33) [inline src/app/shared/messages/auth.messages.ts:3]: Constants must be all capitals.

**Review by Samy-33 — CHANGES_REQUESTED** [review]

File: src/app/shared/services/github.provider.ts (hunk 1 above)
Comment (pranjulps3) [reply src/app/shared/services/github.provider.ts]: Can't do anything. The library exports it this way.

File: src/app/shared/messages/auth.messages.ts (hunk 2 above)
Comment (pranjulps3) [reply src/app/shared/messages/auth.messages.ts:3]: followed the same conventions at most places. Will have to create a separate issue if changes are required.

**Review by Samy-33 — APPROVED** [review]

**Pull request merged**